pandas/core/methods/to_dict.py

```diff
@@ -106,15 +107,16 @@ def to_dict(
         return into_c((k, v.to_dict(into)) for k, v in df.items())
 
     elif orient == "list":
-        object_dtype_indices_as_set = set(box_native_indices)
+        object_dtype_indices_as_set: set[int] = set(box_native_indices)
+
         return into_c(
             (
-                k,
-                list(map(maybe_box_native, v.tolist()))
+                col,
+                list(map(maybe_box_native, df[col].values.tolist()))
```

Comment:
Is `.values` needed here? (and below)

Reply:
yes, this is needed. This is to access the `np.ndarray` data type for the particular column. The `tolist()` method for `np.ndarray` is signnificantly faster that `pd.Series`

Reply:
Interesting what's the performance diff between the two?

Reply:
<img width="1196" alt="Screenshot 2023-08-31 at 2 36 04 AM" src="https://github.com/pandas-dev/pandas/assets/10372220/a466951e-c6c1-4fbc-a532-c82a2532afed">

Reply:
I was referring to specifically `df[col].tolist()` vs `df[col].to_numpy().tolist()`

Reply:
This is `df[col].to_numpy().tolist()`
![Screenshot 2023-08-31 at 11 13 35 PM](https://github.com/pandas-dev/pandas/assets/10372220/e3f32b4c-b92a-4704-a355-4fba5da8c312)
This is `df[col].tolist()`. `list` type orient does not change in performance
![Screenshot 2023-08-31 at 11 14 07 PM](https://github.com/pandas-dev/pandas/assets/10372220/d73a5fa1-c7a3-41af-bb52-5fc2bd852ac8)
This is `df[col].values.tolist()`
![Screenshot 2023-08-31 at 11 14 45 PM](https://github.com/pandas-dev/pandas/assets/10372220/6fa975d2-6455-48a8-92ef-a9c8a3b6dfe3)